File: HoyluShare/app/src/main/java/com/example/zarehhakobian/hoylushare/HoyluDevice.java
package com.example.zarehhakobian.hoylushare;

/**
 * Created by Zareh Hakobian on 01.08.2017.
 */

public class HoyluDevice {
    String hoyluId, name, btAddress, qrValue, nfcValue, publicIp, defaultGateway;

    public HoyluDevice(String hoyluId, String name, String btAddress, String qrValue,
                       String nfcValue, String publicIp, String defaultGateway) {
        this.hoyluId = hoyluId;
        this.name = name;
        this.btAddress = btAddress;
        this.qrValue = qrValue;
        this.nfcValue = nfcValue;
        this.publicIp = publicIp;
        this.defaultGateway = defaultGateway;
    }

    public String getHoyluId() {
        return hoyluId;
    }

    @Override
    public String toString() {
        return this.name;
    }
}

File: HoyluReceiver/HoyluReceiver/HoyluDevice.cs
using System;

namespace HoyluReceiver
{
    public class HoyluDevice
    {
        public HoyluDevice(string name, string hoyluId, string bluetoothAddress, string qrValue, string nfcValue, string publicIp, string defaultGateway)
        {
            Name = name;
            HoyluId = hoyluId;
            BluetoothAddress = bluetoothAddress;
            QrValue = qrValue;
            NfcValue = nfcValue;
            PublicIp = publicIp;
            DefaultGateway = defaultGateway;
        }

        public string Name { get; set; }
        public string HoyluId { get; set; }
        public string BluetoothAddress { get; set; }
        public string QrValue { get; set; }
        public string NfcValue { get; set; }
        public string PublicIp { get; set; }
        public string DefaultGateway { get; set; }

        public DateTime TimestampLastUsed { get; set; } = new DateTime();

        public override string ToString()
        {
            return $"{Name}, am {TimestampLastUsed.ToShortDateString()}";
        }
    }
}
